code_map.py

```diff
@@ -748,6 +778,7 @@ def delay():
                 reset_layout(reduce=True)
 
         sublime.set_timeout_async(delay)
+        sublime.set_timeout_async(focus_source_code)
 
```

Comment:
added this to focus back when closing map with mouse, I didn't have problems for now